components/SubmitPageForm/SubmitPageForm.tsx

```diff
@@ -132,106 +132,110 @@ export const SubmitPageForm = () => {
+                    error={fieldState.error}
+                    value={field.value}
+                    label="Description"
+                    required
+                  />
+                );
+              }}
+            />
+            <Controller
+              name="tags"
+              control={control}
+              render={({ field, fieldState }) => {
+                return (
                   <>
-                    <div className="flex flex-wrap gap-2">
-                      {tags && tags.length > 0 && (
-                        <Tags
-                          className="gap-2 uppercase"
-                          tags={tags}
-                          removeTagHandler={removeTag}
-                        />
-                      )}
-                    </div>
+                    <Input
+                      {...field}
+                      placeholder="Enter your project tags"
+                      onChange={(e) => setTagInput(e.target.value)}
+                      error={fieldState.error}
+                      label="Tags"
+                      value={tagInput}
+                      onKeyDown={(e) => onTagAddition(e, field)}
+                      required
+                      hintMessage="Note: only 5 tags are allowed"
+                    />
+                    {tags && tags.length > 0 && (
+                      <>
+                        <div className="flex flex-wrap gap-2">
+                          {tags && tags.length > 0 && (
+                            <Tags
+                              className="gap-2 uppercase"
+                              tags={tags}
+                              removeTagHandler={removeTag}
+                            />
+                          )}
+                        </div>
+                      </>
+                    )}
                   </>
-                )}
-              </>
-            );
-          }}
-        />
-        <Controller
-          name="content"
-          control={control}
-          render={({ field }) => {
-            return (
-              <RichTextEditor
-                {...field}
-                label={'Content'}
-                value={contentInput}
-                onChange={(value) => onContentChange(value, field)}
-              />
-            );
-          }}
-        />
+                );
+              }}
+            />
+            <Controller
+              name="content"
+              control={control}
+              render={({ field }) => {
+                return (
+                  <RichTextEditor
+                    {...field}
+                    label={'Content'}
+                    value={contentInput}
+                    onChange={(value) => onContentChange(value, field)}
+                  />
+                );
+              }}
+            />
+          </div>
+        </div>
       </form>
       <div className="my-4 w-full">
         <Button
```

Comment:
Place this button in the center of page just like other form elements

Reply:
thanks @rupali-codes for pointing this issue.

@sdifeo the button should be inside the form container width only and it should be right aligned. let us know if you need anything. thanks